src/PrivateDns/PrivateDns/Zones/RemoveAzurePrivateDnsZone.cs

```diff
@@ -0,0 +1,101 @@
+﻿// ------------------------------------------------------------------------------------------------
+// <copyright file="RemoveAzurePrivateDnsZone.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.PrivateDns.Zones
+{
+    using System.Management.Automation;
+    using ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Commands.PrivateDns.Models;
+    using Microsoft.Azure.Commands.PrivateDns.Utilities;
+    using ProjectResources = Microsoft.Azure.Commands.PrivateDns.Properties.Resources;
+
+    /// <summary>
+    /// Deletes an existing zone.
+    /// </summary>
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PrivateDnsZone", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High),OutputType(typeof(bool))]
+    public class RemoveAzurePrivateDnsZone : PrivateDnsBaseCmdlet
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The full name of the zone (without a terminating dot).", ParameterSetName = "Fields")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The resource group in which the zone exists.", ParameterSetName = "Fields")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipeline = true, HelpMessage = "The zone object to set.", ParameterSetName = "Object")]
+        [ValidateNotNullOrEmpty]
+        public PrivateDnsZone Zone { get; set; }
+
+        [Parameter(ParameterSetName = "ResourceId", Mandatory = true, ValueFromPipeline = true, HelpMessage = "Private DNS Zone ResourceID.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not use the ETag field of the Zone parameter for optimistic concurrency checks.", ParameterSetName = "Object")]
```

Comment:
should it be "Does not use"?

Reply:
no this is just sort of a warning message to the user to not use etags.